.gitignore

```diff
@@ -42,3 +42,4 @@ Makefile.dep
 .ccls-cache/*
 compile_commands.json
 redis.code-workspace
+.cache/clangd
```

Comment:
normally we don't ignore files that aren't generated by Redis.

Reply:
`.cache/clangd` is generated by vscode clangd extension, like other lsp-server files.